GVFS/GVFS.Virtualization/FileSystemCallbacks.cs

```diff
@@ -768,6 +807,26 @@ private FileSystemTaskResult ExecuteBackgroundOperation(FileSystemTask gitUpdate
             return result;
         }
 
+        private FileSystemTaskResult TryRemoveModifiedPath(string virtualPath, bool isFolder)
+        {
+            string fullPathToItem = Path.Combine(this.context.Enlistment.WorkingDirectoryRoot, virtualPath);
+            if ((isFolder && this.context.FileSystem.DirectoryExists(fullPathToItem)) || 

```

Comment:
Nit: I would remove the redundant parens. To me, they actually make it harder to read.

Reply:
Removing these caused a style cop error.